### PR TITLE
Changing tests

### PR DESCRIPTION
**Pull request by tsenapathi**

### CODE REVIEW

File: tools/mdanalysis/rdf.xml [hunk 1]
```diff
@@ -51,10 +51,10 @@
             <param name="start" value="0.0"/>
             <param name="end" value="5.0"/>
             <output name="output">
-              <assert_contents>
-                <has_n_columns n="2" />
-              </assert_contents>
-             </output>
+             <assert_contents>
```
Comment (bgruening) [inline tools/mdanalysis/rdf.xml]: you could add additional asserts here

**Comment by bgruening** [comment]
Thanks @tsenapathi! Its strange that my commits are here as well, can you rebase this branch against master?

**Comment by KJNaidoo** [comment]
Excellent,  It looks like this issue is resolved for publication.

File: tools/mdanalysis/test-data/RDF_raw_data.tabular [hunk 2]
```diff
@@ -1,100 +0,0 @@
-0.025	0.0
-0.075	0.0
```
Comment (bgruening) [inline tools/mdanalysis/test-data/RDF_raw_data.tabular]: @tsenapathi it looks like travis is returning a different result in which the second column is `nan`.
```
+3.825	nan
+3.875	nan
+3.925	nan
+3.975	nan
+4.025	nan
+4.075	nan
+4.125	nan
+4.175	nan
```

These are exactly the kind of error the reviewer wanted to catch I assume. It seems to me like a valid error or a difference in requirements.

Comment (tsenapathi) [reply tools/mdanalysis/test-data/RDF_raw_data.tabular]: @bgruening @chrisbarnettster is this a problem with travis or the tool? any ideas how to solve this?

Comment (chrisbarnettster) [reply tools/mdanalysis/test-data/RDF_raw_data.tabular]: @tsenapathi . Assert that it is not nan?

Comment (chrisbarnettster) [reply tools/mdanalysis/test-data/RDF_raw_data.tabular]: A very simplistic test ...

    <assert_contents>
        <has_n_columns n="2" />
        <not_has_text text="nan" />
    </assert_contents>

Comment (chrisbarnettster) [reply tools/mdanalysis/test-data/RDF_raw_data.tabular]: For this specific dataset the outcomes are known. Even with minor changes in precision we should be able to match the output using 'has_line_matching '
For example this line "3.275	6336.434022284689" can be matched with 
<has_line_matching expression="3.275\s+6336.43" />.
This is only for ensuring the outcome for this particular test data set.

Comment (chrisbarnettster) [reply tools/mdanalysis/test-data/RDF_raw_data.tabular]: not sure why nan. 0.0 makes sense because this is such a short trajectory.

Comment (chrisbarnettster) [reply tools/mdanalysis/test-data/RDF_raw_data.tabular]: see example assert (will need minor adjustment) used here https://github.com/usegalaxy-eu/workflow-testing/pull/40/commits/244fba640e318df5808df403dd5c92e245bb0e4e

**Comment by bgruening** [comment]
@tsenapathi let me know when you need a review.

**Comment by tsenapathi** [comment]
Hi, @bgruening and @chrisbarnettster ,
See my latest commits to change tests, I have added more assert tests and also cleaned up few more other issues. Please have a look at this.

**Comment by tsenapathi** [comment]
@bgruening you can review it now 😃

**Comment by bgruening** [comment]
Merci!

**Pull request merged**